lib/removeUnusedVertices.js

```diff
@@ -179,10 +188,15 @@ function calculateIndexIntervals(accessors, buffers, bufferViews, meshes) {
                                 var attributeView = bufferViews[attributeAccessor.bufferView];
 
                                 if (Object.keys(attributeView.extras._pipeline.accessors).indexOf(attributes[attributeId]) === -1) {
-                                    var attributeStride = 1;
+                                    var attributeStride = 1; // Default to width of BYTE and UNSIGNED BYTE
                                     if (attributeAccessor.byteStride > 0) {
                                         attributeStride = attributeAccessor.byteStride;
                                     }
+                                    else if (defined(attributeAccessor.type)){
+                                        // If an attribute type is defined, a byteStride of 0 is equivalent to a byteStride
+                                        // of BYTE_COUNT(type). The attributeStride is the width of the attribute type.
+                                        attributeStride = getAccessorByteStride(attributeAccessor);
+                                    }
                                     else {
                                         //Int16
                                         if (attributeAccessor.componentType === WebGLConstants.SHORT || attributeAccessor.componentType === WebGLConstants.UNSIGNED_SHORT) {

```

Comment:
Could this whole section:

```javascript
var attributeStride = 1; // Default to width of BYTE and UNSIGNED BYTE
if (attributeAccessor.byteStride > 0) {
    attributeStride = attributeAccessor.byteStride;
}
else if (defined(attributeAccessor.type)){
    // If an attribute type is defined, a byteStride of 0 is equivalent to a byteStride
    // of BYTE_COUNT(type). The attributeStride is the width of the attribute type.
    attributeStride = getAccessorByteStride(attributeAccessor);
}
else {
    //Int16
    if (attributeAccessor.componentType === WebGLConstants.SHORT || attributeAccessor.componentType === WebGLConstants.UNSIGNED_SHORT) {
        attributeStride = 2;
    }
    //Float32
    if (attributeAccessor.componentType === WebGLConstants.FLOAT) {
        attributeStride = 4;
    }
}
```

be replaced by

```javascript
var attributeStride = getAccessorByteStride(attributeAccessor);
```

The accessor's `type` is required right? If this isn't a safe assumption, then we could modify `getAccesorByteStride` or `numberOfComponentsForType` to account for an undefined type instead of checking here.